Package.swift

```diff
@@ -4,120 +4,170 @@
 import PackageDescription
 import CompilerPluginSupport
 
-// Availability Macros
-let availabilityMacros: [SwiftSetting] = [
```

Comment:
@parkera  This might have to default to be `true` otherwise tests won't run on CI... or maybe we should set some special environment variables for testing?

Reply:
Hm, how does the swift-crypto project do this? We should certainly resolve before merging because we don't want to effectively disable building for macOS in CI.